internal/cobraext/flags.go

```diff
@@ -65,6 +65,21 @@ const (
 	BenchCorpusRallyUseCorpusAtPathFlagName        = "use-corpus-at-path"
 	BenchCorpusRallyUseCorpusAtPathFlagDescription = "path of the corpus to use for the benchmark: if present no new corpus will be generated"
 
+	BenchStreamBackFillFlagName        = "backfill"
+	BenchStreamBackFillFlagDescription = "amount of time to ingest events for since starting from now: expressed as a negative duration"
```

Comment:
The description is very clear, but of course I put in `1m` at first.

Reply:
It is indeed a bit weird to have a parameter that only accepts negative numbers. Could we revert it, so it accepts only positive numbers and we negate it in code?